### DISCUSSION TITLE
SSML

### DISCUSSION THREAD

**Discussion by House-martin** [pgmichael/wavenet-for-chrome#51] [Q&A]
"SSML support for text under 5000 characters", but how do I use an ssml file?
I've tried putting:
<speak>speak</emphasis> </speak><break strength="x-strong"/> these words<break strength="weak"/> </speak>
in Notepad++, saved it as html, opened in Chrome.  I must be doing something wrong.

**Comment by pgmichael** [comment]
Any text that starts with a `<speak>` and ends with a `</speak>` tag will be considered as an SSML input. 

If you open up a file as `.html` inside your browser, the markup won't be displayed onscreen and therefore not part of the selection. 

What you can do is save it as a regular `.txt` file. This way the markup will beshown on the page and therefore part of the selection.

Next version will have a "Sandbox" view inside the popup in which you can paste an playback text. Hopefully this will make this kind of use case less troublesome.
